connectors/sources/outlook.py

```diff
@@ -832,6 +835,7 @@ async def download_func(self, content):
 
     async def _fetch_attachments(self, attachment_type, outlook_object, timezone):
         for attachment in outlook_object.attachments:
+            self._logger.debug(f"Fetching attachment for {attachment.name}")
```

Comment:
It's good to log the attachment name, but we need to log what outlook_object we're fetching attachment too, since afaik the connector is fetching data for multiple hierarchies at the same time.

E.g.:

`Fetching attachment "test_plan.xls" for calendar record.` or something, ideally indicating what the parent object is, so it would be possible to correlate this line to other lines you added.

Reply:
Got it. I've added a parent object in the log message, here `attachment_type` indicates the parent object of the attachment.